### PR TITLE
6474 cbyrd ensure all template strings are i18n wrapped

### PR DESCRIPTION
**Pull request by chrabyrd**
### Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change
Manually ensure all template strings are i18n wrapped.

Testing doc here: https://docs.google.com/spreadsheets/d/1Is4sbq-UPAWBypff0EnQalvmaTX0C9XO3I3XqLW46hY/edit?usp=sharing

### Issues Solved

#6474 

### Checklist

-   [x] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by chrabyrd** [comment]
First pass finished, all strings should be wrapped.

File: arches/app/templates/login.htm [hunk 1]
```diff
@@ -75,7 +75,7 @@ <h4>{% trans "Login failed" %}</h4>
 			</div>
 
 			<div class="panel-footer">
-				Learn more about <a href="archesproject.org">{% trans "Arches" %}</a>
+				{% trans "Learn more about" %} <a href="archesproject.org">{% trans "Arches" %}</a>
```
Comment (apeters) [inline arches/app/templates/login.htm:78]: this whole line including tags should be wrapped in a "trans" tag, this way the translator can put the emphasis in the correct context
```{% trans "Learn more about <a href='archesproject.org'>Arches</a>" %}```

File: arches/app/templates/registration/password_reset_form.html [hunk 2]
```diff
@@ -24,6 +24,6 @@ <h3 class="login-panel-header arches-signin">{% trans "Password Recovery" %}</h3
 
 	</form>
 	<div class="panel-footer">
-        Learn more about <a href="archesproject.org">{% trans "Arches" %}</a>
+        {% trans "Learn more about" %} <a href="archesproject.org">{% trans "Arches" %}</a>
```
Comment (apeters) [inline arches/app/templates/registration/password_reset_form.html:27]: this whole line including tags should be wrapped in a "trans" tag, this way the translator can put the emphasis in the correct context
```{% trans "Learn more about <a href='archesproject.org'>Arches</a>" %}```

File: arches/app/templates/signup.htm [hunk 3]
```diff
@@ -150,7 +150,7 @@ <h3 class="login-panel-header arches-signin">{% trans "Create Account" %}</h3>
             </div>
 
             <div class="panel-footer">
-                Learn more about <a href="archesproject.org">{% trans "Arches" %}</a>
+                {% trans "Learn more about" %} <a href="archesproject.org">{% trans "Arches" %}</a>
```
Comment (apeters) [inline arches/app/templates/signup.htm:153]: this whole line including tags should be wrapped in a "trans" tag, this way the translator can put the emphasis in the correct context
```{% trans "Learn more about <a href='archesproject.org'>Arches</a>" %}```

File: arches/app/templates/views/graph/graph-designer/permission-settings-form.htm [hunk 4]
```diff
@@ -14,13 +14,13 @@
         <div class="permissions-instructions-panel">
             <span class="mpm-instruction-title">{% trans "To set permissions: " %} </span>
             <div class="mpm-instruction">
-                <strong> {% trans "1. Select one or more cards" %}</strong> from the card tree
+                <strong> {% trans "1. Select one or more cards" %}</strong> {% trans "from the card tree" %}
```
Comment (apeters) [inline arches/app/templates/views/graph/graph-designer/permission-settings-form.htm:17]: this whole line should be wrapped in a single trans tag

File: arches/app/templates/views/graph/graph-designer/permission-settings-form.htm [hunk 5]
```diff
@@ -14,13 +14,13 @@
...
             </div>
             <div class="mpm-instruction">
-                <strong> {% trans "2. Select a Group or User Account" %}</strong> from the dropdown below.
+                <strong> {% trans "2. Select a Group or User Account" %}</strong> {% trans "from the dropdown below." %}
```
Comment (apeters) [inline arches/app/templates/views/graph/graph-designer/permission-settings-form.htm:20]: this whole line should be wrapped in a single trans tag

File: arches/app/templates/views/graph/graph-designer/permission-settings-form.htm [hunk 6]
```diff
@@ -14,13 +14,13 @@
...
             </div> 
             <div class="mpm-instruction">
-                <strong> {% trans "3. Apply Permissions" %}</strong> to set your changes.  You'll see your selections reflected by the icons in the card tree.
+                <strong> {% trans "3. Apply Permissions" %}</strong> {% trans "to set your changes.  You'll see your selections reflected by the icons in the card tree." %}
```
Comment (apeters) [inline arches/app/templates/views/graph/graph-designer/permission-settings-form.htm:23]: this whole line should be wrapped in a single trans tag

File: arches/app/templates/views/graph/graph-settings.htm [hunk 7]
```diff
@@ -154,7 +154,7 @@ <h3 class="panel-title card-main-title">{% trans "Settings" %}</h3>
                                                         </div>
 
                                                         <div class="col-xs-12">
-                                                            <select class="design" {% if node_count > 1 %}disabled{% endif %} data-bind="value: graph.ontology_id, valueAllowUnset: true, options: ontologies, optionsText: 'name', optionsValue:'ontologyid', optionsCaption: '{% trans "No ontology" %}', chosen: {disable_search_threshold: 10, width: '100%'}"></select>
+                                                            <select class="design" {% if node_count > 1 %}disabled{% endif %} data-bind="value: graph.ontology_id, valueAllowUnset: true, options: ontologies, optionsText: {% trans 'name' %}, optionsValue:'ontologyid', optionsCaption: {% trans 'No ontology' %}, chosen: {disable_search_threshold: 10, width: '100%'}"></select>
```
Comment (apeters) [inline arches/app/templates/views/graph/graph-settings.htm:157]: **optionsText** should NOT be wrapped in a trans tag, as this is a keyword in code

File: arches/app/templates/views/graph/graph-settings.htm [hunk 8]
```diff
@@ -166,7 +166,7 @@ <h3 class="panel-title card-main-title">{% trans "Settings" %}</h3>
                                                         </div>
 
                                                         <div class="col-xs-12">
-                                                            <select class="design" {% if node_count > 1 %}disabled{% endif %} data-bind="value: ontologyClass, valueAllowUnset: false, options: ontologyClasses, optionsText: 'display', optionsValue:'source', chosen: {disable_search_threshold: 10, width: '100%', search_contains: true}"></select>
+                                                            <select class="design" {% if node_count > 1 %}disabled{% endif %} data-bind="value: ontologyClass, valueAllowUnset: false, options: ontologyClasses, optionsText: {% trans 'display' %}, optionsValue:'source', chosen: {disable_search_threshold: 10, width: '100%', search_contains: true}"></select>
```
Comment (apeters) [inline arches/app/templates/views/graph/graph-settings.htm:169]: **optionsText** should NOT be wrapped in a trans tag, as this is a keyword in code

File: arches/app/templates/views/graph/permission-manager/permission-settings-form.htm [hunk 9]
```diff
@@ -42,7 +42,7 @@ <h3 id="cc-title" class="panel-title card-main-title">{% trans "Permissions" %}<
             <div class="tab-base card-content-container pad-top">
                 <div>
                     <p class="mpm-instruction">
-                        To set permissions: <strong> Click on the card(s) </strong>you want from the list on the left, then select the permission levels from the form on the right.
+                        {% trans "To set permissions:" %} <strong> {% trans "Click on the card(s)" %} </strong>{% trans "you want from the list on the left, then select the permission levels from the form on the right." %}
```
Comment (apeters) [inline arches/app/templates/views/graph/permission-manager/permission-settings-form.htm:45]: this whole line should be wrapped in a single trans tag

**Pull request merged**